packages/backend/src/database/migrations/20250403090130_add_nested_paces_ability.ts

```diff
@@ -0,0 +1,50 @@
+import { Knex } from 'knex';
+
+const tableName = 'spaces';
+
+export async function up(knex: Knex): Promise<void> {
+    await knex.raw('CREATE EXTENSION IF NOT EXISTS ltree');
```

Comment:
to discuss

Reply:
so it looks like this in in the standard library, so no problem: https://www.postgresql.org/docs/current/ltree.html